documentation/ADRs/009_log_file_for_generated_data.md

```diff
@@ -0,0 +1,49 @@
+# Log file for generated data
+
+
+| ADR Info            | Details                     |
+|---------------------|-----------------------------|
+| Subject             | Log file for generated data |
+| ADR Number          | 009                         |
+| Status              | Accepted                    |
+| Author              | Xiaolong                    |
+| Date                | 09.09.2024                  |
+
+## Context
+In the context of the VIEWS pipeline, there is a need to create a log file to ensure that models and data are tracked accurately and meet certain criteria before running. 
```

Comment:
I think we should add that these log files are used as metadata for deployment and orchestration (maybe even allude to how)